source/shared/net/guides/guide-ledger.rst

```diff
@@ -0,0 +1,148 @@
+
+#. In the Desktop Wallet, there’s a message asking you to open the Concordium application on the Ledger. On the Ledger, press both buttons when it says **Concordium**. The Ledger says **Concordium is ready**.
+
+#. In the Desktop Wallet, there’s a message saying the Ledger is ready, depending. Select **Submit**.
+
+#. Press both buttons to confirm the following on the Ledger:
+
+   - **Create credential**: the number that is assigned to the credential is displayed.
+
+   - **Public key**: verify that the public key on the Ledger corresponds to the public key in the Desktop Wallet. Use the right button to navigate through the key.
+
+#. In the Desktop Wallet, select **Continue**.
+
+#. The Ledger says **Review identity provider info**. Press both buttons, and then press the right button to navigate through the public key and verify that it corresponds to the information in the Desktop Wallet. Press both buttons to confirm.
+
+#. Verify that the **Signature threshold** on the Ledger corresponds to the threshold in the Desktop Wallet.
+
+#. Press the right button, and then press both buttons to sign the identity provider information.
+
+#. In the Desktop Wallet, the **New identity** page is displayed. Enter the identity verification information and select **Submit**. This information will vary depending on the identity provider. The identity provider submits the identity to the blockchain. When the initial account is created in a finalized block on the blockchain, the identity provider confirms the identity, and after confirmation, a green checkmark is displayed next to the Concordium logo on the identity.
+
+#. Select **Finished**. When you’ve created more accounts, you can recognize your initial account by the text *identity* next to the name.
+
+View the account balance
+========================
+
+To view the account balance in the Concordium Desktop Wallet do the following:
+
+- In the Desktop Wallet, go to **Accounts**, and then select the account whose balance you want to see. You can now see the following account information:
+
+  - **Account Total** shows the total number of GTU on the account
+
+  - **Balance** shows the GTU that are at your disposal, the amount of GTU that you’ve staked, and any GTU that are locked in a release schedule.
+
+  - **Shielded balance** shows the amount of GTU that you’ve shielded.
+
+  - **Staked** shows the amount of GTU you've staked if you're node is :ref:`baking <baker-concept>` blocks.
+
+    .. image:: ../images/desktop-wallet/account-balance.png
+         :width: 60%
+
+For more information about accounts, see :ref:`Overview of accounts <overview-account-desktop>`.
+
+How to receive GTU
+==================
+
+Currently, you don’t have to use the Ledger to receive GTU. To see any transfers to your account do the following:
```

Comment:
Would it make sense to mention how to find your account address in this section?